src/lib/eliom_form_sigs.shared.mli

```diff
@@ -0,0 +1,1034 @@
+type input_type =
+  [ `Url
+  | `Tel
+  | `Text
+  | `Time
+  | `Search
+  | `Password
+  | `Checkbox
+  | `Range
+  | `Radio
+  | `Submit
+  | `Reset
+  | `Number
+  | `Hidden
+  | `Month
+  | `Week
+  | `File
+  | `Email
+  | `Image
+  | `Datetime_local
+  | `Datetime
+  | `Date
+  | `Color
+  | `Button
+  ]
+
+type button_type =
+  [ `Button
+  | `Reset
+  | `Submit
+  ]
+
+module type PARAM = sig
+
+  type +'a elt
+  type +'a attrib
+
+  type uri
+
+  val a_input_required : unit -> Html5_types.input_attrib attrib
+
+  val a_select_required : unit -> Html5_types.select_attrib attrib
```

Comment:
And if we don't (it's not merged in tyxml yet ..). Just put the `` `Required`` argument back